unlock-protocol.com/blog/tokenizing-memberships.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Tokenizing Memberships
+subTitle: What if there was a new way to interact with the communities you are part of?
+authorName: Julien Genestoux
+publishDate: October 18, 2019
+description: What if there was a new way to interact with the communities you are part of?
+image: /static/images/blog/tokenized-memberships.jpg
+---
+
+The web needs a [new business model](https://medium.com/unlock-protocol/its-time-to-unlock-the-web-b98e9b94add1) — one which doesn’t erode people’s privacy or centralize control into the hands of a few powerful entities.
+
+Luckily, the web is resilient, it’s reliable, and it’s really, really good at lots of things. Here’s one example: it’s great at creating communities.
```

Comment:
```suggestion
Luckily, the web is resilient, reliable, and really, really good at lots of things. Here’s one example: it’s great at creating communities.
```

Removing the "it's"s really gets that alliteration going. Also we may want a better way to say "good at lots of things."